test/functional/sqlsrv/sqlsrv_complex_query.phpt

```diff
@@ -1,23 +1,69 @@
 --TEST--
 Test a complex query with IDENTITY_INSERT
+--DESCRIPTION--
+Verifies the behavior of INSERT queries with and without the IDENTITY flag set.
+This test is similar to the other test TC33_ComplexQuery but using UTF-8 data
+--SKIPIF--
+<?php require('skipif_versions_old.inc'); ?>
 --FILE--
 ﻿<?php
 require_once('MsCommon.inc');
 
-function ComplexQuery()
+function insertTest($conn, $tableName, $inputs, $query)
```

Comment:
since query isn't null, you don't need to check for `is_null($query)` in the function